creator-node/src/apiHelpers.js

```diff
@@ -66,6 +66,7 @@ module.exports.handleResponseWithHeartbeat = (func) => {
 }
 
 const sendResponse = (module.exports.sendResponse = (req, res, resp) => {
+  const skipErrorLogStatusCodes = [403, 404] // skip logging response error
```

Comment:
i think this is the change i'm most on the fence about. this removes a ton of error logs like "cannot be served by this node",  "No valid file found for provided dirCID", "File with path not found" etc. However trying to weigh usefulness vs necessity

Reply:
the compromise here is we include these as debug logs on prod

Reply:
I feel like we shouldn't remove these logs bc they give us insight on why flows fail

Reply:
i can keep this in for now. we can always prune later

Reply:
not sure about this change, can handle separately